client/__Tests__/App.test.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { shallow, mount, render } from 'enzyme';
+import Carousel from '../components/Carousel.jsx';
+
+const wrapper = shallow(<Carousel />)
+
+describe('DOM render', ()=>{
+  it('Should render Carousel to the DOM', () => {
+    expect(wrapper.find('.carousel').exists()).toBe(true)
+  })
```

Comment:
You should install a linter immediately so that your tooling can tell you about missing semicolons instead of your PR reviewers.